libs/feature/record/src/lib/data-view-map/data-view-map.component.ts

```diff
@@ -121,7 +121,10 @@ export class DataViewMapComponent {
         type: MapContextLayerTypeEnum.WMS,
         name: link.name,
       })
-    } else if (link.protocol.startsWith('OGC:WFS')) {
+    } else if (
+      link.protocol.startsWith('OGC:WFS') ||
```

Comment:
I would put all this kind of checks in helpers

Reply:
Good point, I'll adapt that

Reply:
So I finally put them in the `link.helper`, but indeed we might do some more refacto/merging of the three link services